backend/src/api/courses/courses.api.ts

```diff
@@ -115,6 +123,33 @@ const initCoursesApi: FastifyPluginAsync<Options> = async (fastify, opts) => {
       return rep.status(HttpCode.OK).send(course);
     },
   });
+
+  fastify.route({
+    method: HttpMethod.POST,
+    url: CoursesApiPath.$ID_MENTORS,
+    schema: {
+      params: courseGetParamsValidationSchema,
+      body: courseMentorCreateBodyValidationSchema,
+    },
+    async handler(
+      req: FastifyRequest<{
+        Params: CourseSelectMentorRequestParamsDto;
+        Body: CourseSelectMentorRequestDto;
+      }>,
+      rep,
+    ) {
+      const { mentorId, menteeId } = req.body;
+      const { id } = req.params;
+      const menteeToMentor =
+        await menteesToMentorsService.createMenteesToMentors({
```

Comment:
```suggestion
        await mentorService.createMenteesToMentors({
```
menteesToMentorsService should be inside mentor service